scheduler/src/cook/mesos/util.clj

```diff
@@ -35,7 +73,7 @@
           db)
        (map first)))
 
-(defn categorize-job
+(defn- categorize-job-miss
```

Comment:
Move this into a `let-fn` inside categorize-job.

Reply:
I thought we favor `let` over `let-fn`